HW17/tests/test_checkout_page.py

```diff
@@ -0,0 +1,43 @@
+from pytest import mark
+
+
+@mark.regression
+def test_user_can_not_proceed_with_empty_inputs(open_checkout_page):
+    checkout_page = open_checkout_page
+    checkout_page.click_continue_button()
+    assert checkout_page.check_presence_error_state(), \
+        'Something went wrong. Error state not returned.'
```

Comment:
So instead of getting an error message from your page in assertion error, you will get False. It is not informative

After fixing page objects you should fix all tests all around project